tutorials/connect-jdbc-kstreams-rekey-specificavro/kstreams.html

```diff
@@ -1,6 +0,0 @@
----
-layout: tutorial
-permalink: /connect-jdbc-kstreams-rekey-specificavro/kstreams
-stack: kstreams
-static_data: connect-jdbc-kstreams-rekey-specificavro
----
```

Comment:
We need to preserve this file so that existing links are redirected. Content can be changed to:
```
---
permalink: /connect-jdbc-kstreams-rekey-specificavro/kstreams	
canonical: connect-add-key-to-source/kstreams.html
---

<!DOCTYPE html>	
<html>	
  <head>	
    <meta http-equiv="refresh" content="0; url={{ page.canonical | absolute_url }}">	
    <link rel="canonical" href="{{ page.canonical | absolute_url }}" />	
  </head>	
  <body></body>	
</html>
```

Reply:
Done